support-frontend/assets/components/marketingConsent/marketingConsentWithCheckbox.jsx

```diff
@@ -112,7 +113,25 @@ class MarketingConsentWithCheckbox extends Component<PropTypes, StateTypes> {
   }
 
   render() {
-    const emailFrequency = this.props.countryGroupId === 'AUDCountries' ? 'an occasional' : 'a weekly';
+    const createTitleText = () => {
+      const { countryGroupId, contributionType } = this.props;
+      const titleTexts = {
+        UsAllPlusRowRecurring: 'a weekly email',
+        AuAll: 'an occasional email',
+        UkRowSingle: 'occasional emails',
+      };
+
+      if (countryGroupId === 'UnitedStates') {
+        return titleTexts.UsAllPlusRowRecurring;
```

Comment:
Please confirm my thinking here - everyone in the US has the option to receive a weekly email even if they've made a single contribution?

Reply:
Iona has clarified - all single will have same 'occasional' messaging